packages/codemirror-lang-kcl/src/kcl.grammar

```diff
@@ -59,7 +59,9 @@ UnaryOp { AddOp | BangOp }
 
 ObjectProperty { PropertyName (":" | Equals) expression }
 
-ArgumentList { "(" commaSep<expression> ")" }
+NamedArgument { ArgumentName Equals expression }
```

Comment:
nit: In the code, we've called these labeled arguments, not named arguments. It'd be good to keep the terminology consistent.